exhibitions/src/main/kotlin/edu/artic/exhibitions/AllExhibitionsFragment.kt

```diff
@@ -45,6 +47,33 @@ class AllExhibitionsFragment : BaseViewModelFragment<AllExhibitionsViewModel>()
                 .disposedBy(disposeBag)
     }
 
+    override fun setupNavigationBindings(viewModel: AllExhibitionsViewModel) {
+        viewModel.navigateTo.subscribe {
+            when (it) {
+
+                is Navigate.Forward -> {
+                    when(it.endpoint) {
+
+                        is AllExhibitionsViewModel.NavigationEndpoint.ExhibitionDetails -> {
+                            val endpoint = it.endpoint as AllExhibitionsViewModel.NavigationEndpoint.ExhibitionDetails
+                            val view = recyclerView.findViewHolderForAdapterPosition(endpoint.pos).itemView.image
+                            fragmentManager?.let { fm ->
```

Comment:
we should coordinate switch to navigation library here.

Reply:
yea this was before we had any navigation library so it was to be updated at a later point once nav was there.